constraints_gpu.txt

```diff
@@ -39,7 +39,7 @@ exceptiongroup==1.2.2
 fastapi==0.115.3
 filelock==3.16.1
 flatbuffers==24.3.25
-flax==0.10.6
+flax==0.10.7
```

Comment:
Please also update in `requirements.txt` and `requirements_with_jax_ai_image.txt`